notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1219 @@
+{
```

Comment:
<p>Grammar nits:</p><p> </p><p>This notebook demonstrates how <s>you</s> <strong>to</strong> ingest, analyze, and write <s>back </s>data to BigQuery using Apache Spark <s>on</s> with <a href="https://cloud.google.com/dataproc" target="_blank">Dataproc<s> Cluster</s></a><s>.</s> Using <s>the</s> GitHub Activity Data, <s>Y</s><strong>y</strong>ou will analyze repositories in GitHub and find out what kind of programming languages <strong>are</strong> being used in their repositories.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>